minio/policy.py

```diff
@@ -528,36 +528,50 @@ def _get_permissions(s, resource, object_resource, matched_resource,
 
 
 # Returns policy of given bucket name, prefix in given statements.
-def get_policy(statements, bucket_name, prefix=''):
```

Comment:
Can you tell me problems in this function?  I am not able to understand why you need to change this.